extensions/wikia/SpitfiresContributionExperiments/templates/MyProfileTourExperimentStep3.mustache

```diff
@@ -0,0 +1,8 @@
+<div class="my-profile-step step-3"></div>
+<a class="my-profile-header-go-back">BACK</a>
+<h3 class="my-profile-welcome-header">Your profile is almost ready!</h3>
+<form class="my-profile-form">
+	<label class="my-profile-label" for="my-profile-gaming-platforms">Tell us a few words about yourself:</label>
+	<textarea class="my-profile-textarea" id="my-profile-gaming-platforms">{{answer}}</textarea>
```

Comment:
Same ID as the previous one about gaming platforms.

Also, do we need to load four different templates that are basically the same, or can we load one template and have the header and messages be part of the config that is passed to the template?